### ISSUE TITLE
Don't ignore second argument of `\setkomavar`

### ISSUE THREAD

**Issue by stakaz** [valentjn/vscode-ltex#373]
Hello, is it currently possibly to mix checking for different arguments, e.g. only check the second argument in `\setkomavar{fistname}{Maxxxx}`? 

Maybe I am missing something here, but if not, such a feature would be great.

Thanks.

**Comment by valentjn** [comment]
The problem is `\setkomavar{}{}` being ignored by default. Something like

```json
{
  "ltex.latex.commands": {
    "\\setkomavar{}": "ignore",
    "\\setkomavar{}{}": "default"
  }
}
```

should work, but it doesn't due to a bug.

**Comment by stakaz** [comment]
Ok, but I right, that it it not possible by default to mix different settings for different arguments? That would be good, in order to do stuff like this, e.g. `\mycustomheading{Title which should be checked}{colorNotToCheck}`. I think, you get the point. I mean, each possibility like ignore dummy and so on works on its own, so it would be a "minor change", to something like: 

```
{
  "ltex.latex.commands": {
    "\\setkomavar{}{}": {"ignore", "dummy"}
  }
}
```

**Comment by valentjn** [comment]
It would definitely not be a &ldquo;minor change,&rdquo; as this would require a complete rewrite of the command matching system. PRs are appreciated, though.

I don't see a real-world example that's used widely, which is a necessary condition for feature requests. Your example with `\setkomavar` is covered&mdash;or it isn't, due to the bug. Therefore, I'll turn this issue into a bug report.

**Comment by stakaz** [comment]
Ok, thanks for the explanations. A appreciate your work and "easy" was probably the wrong word anyway.

**Comment by valentjn** [comment]
Fixed in 11.0.0.
